lib/compute_runtime/cli.rb

```diff
@@ -12,8 +12,9 @@ def run_rbwasm(*args)
       RubyWasm::CLI.new(stdout: $stdout, stderr: $stderr).run(args)
     end
 
-    def pack_directory(infile, outfile, dirmaps)
+    def pack_directory(infile, outfile, dirmaps, dirmap_strings)
       mapdir_args = dirmaps.collect_concat { |k, v| ["--mapdir", "#{k}::#{v}"] }
+      mapdir_args += dirmap_strings.collect_concat { |str| ["--mapdir", str] }
```

Comment:
```suggestion
      mapdir_args += dirmap_strings.collect_concat { |str| ["--dir", str] }
```

Note that `--mapdir GUEST_DIR::HOST_DIR` is old one and `--dir HOST_DIR::GUEST_DIR` have the opposite mapping direction.